### PR TITLE
Kosaraju Algorithm Code in C++ added

### LINKED ISSUE #445: Kosaraju’s algorithm
Hey, I am Saksham Sharma, a GSSOC'21 participant. I am a Competitive Programmer and C++ enthusiastic.
I want to write **Kosaraju's algorithm** to find the strongly connected components in a graph in C++.

Please assign this task to me.

@anubhavitis @dheerajkotwani  @tarun26091999  @todi-2000 @geekquad @plazzy99 @rudrakshi99

### PR DESCRIPTION
**Pull request by ssaksham**
Hey, This is GSSOC21 participant and this PR closes #445 .

### CODE REVIEW

**Comment by ssaksham** [comment]
@plazzy99 @rudrakshi99 @nakul-19 @tarun26091999 @OjusWiZard @shreyanspoddar @raksha009 @anjul2000 Please review my PR. I have created it 11 days ago.

File: Code/C++/kosaraju_algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+6 7
+7 5
+8 6
+8 9
+9 8
+9 10
+
+Output 2:
+2 3 4 1
+6 7 5
```
Comment (OjusWiZard) [inline Code/C++/kosaraju_algorithm.cpp:201]: Output isn't exactly the same as it's coming from this code. Although, this one is also correct, still write the same output.

**Review by OjusWiZard — CHANGES_REQUESTED** [review]
Link the code file to the appropriate Readme

File: Code/C++/kosaraju_algorithm.cpp (hunk 1 above)
Comment (ssaksham) [reply Code/C++/kosaraju_algorithm.cpp]: Done!

**Comment by ssaksham** [comment]
@OjusWiZard Please review.

File: Code/C++/kosaraju_algorithm.cpp (hunk 1 above)
Comment (OjusWiZard) [reply Code/C++/kosaraju_algorithm.cpp]: It's still not the same...

**Review by OjusWiZard — APPROVED** [review]
well done

**Review by OjusWiZard — CHANGES_REQUESTED** [review]
Also, link the Code to the appropriate readme

**Comment by ssaksham** [comment]
Done!

**Review by plazzy99 — APPROVED** [review]
LGTM 👍🏻

**Review by OjusWiZard — APPROVED** [review]

**Pull request merged**